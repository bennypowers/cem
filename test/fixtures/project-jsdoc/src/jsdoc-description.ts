/**
 * description
 */
@customElement('jsdoc-description')
class JsdocDescription extends LitElement { }
