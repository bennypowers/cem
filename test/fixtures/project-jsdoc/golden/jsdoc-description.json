{
  "schemaVersion": "1.0.0",
  "modules": [
    {
      "kind": "javascript-module",
      "path": "src/jsdoc-description.js",
      "declarations": [
        {
          "name": "JsdocDescription",
          "description": "description",
          "superclass": {
            "name": "LitElement",
            "package": "lit"
          },
          "kind": "class",
          "tagName": "jsdoc-description",
          "customElement": true
        }
      ],
      "exports": [
        {
          "kind": "custom-element-definition",
          "name": "jsdoc-description",
          "declaration": {
            "name": "JsdocDescription",
            "module": "/var/home/bennyp/Developer/cem/test/fixtures/project-jsdoc/src/jsdoc-description.js"
          }
        }
      ]
    }
  ]
}